components/service/fretboard/src/main/java/mozilla/components/service/fretboard/SyncResult.kt

```diff
@@ -0,0 +1,13 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.service.fretboard
+
+/**
+ * Represents an experiment sync result
+ */
+data class SyncResult(
```

Comment:
Let's try to find a different name. `Sync` is usually associated with Firefox Sync and that may get confusing. How about something like `ExperimentsSnapshot`?

And then in the code it's `snapshot.experiments` and `snapshot.lastModified`?

Reply:
Looks good!